lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -42,10 +42,10 @@ def all_questions
     end
 
     def region(postcode)
-      is_london?(postcode) ? :london : :national
+      london?(postcode) ? :london : :national
     end
 
-    def is_london?(postcode)
+    def london?(postcode)
```

Comment:
I found it slightly surprising that no changes to the flow were required in this commit. That appears to be because we use `region == :london` in the flow, i.e. the `#london?` method is only used within this class. As such it might be clearer if it was made private. Alternatively we could use the method in the conditionals within the flow. However, presumably that would mean extra requests to the external Imminence endpoint.